cli-output/src/cli_output.rs

```diff
@@ -1532,55 +1532,28 @@ impl fmt::Display for CliProgramAuthority {
 #[serde(rename_all = "camelCase")]
 pub struct CliUpgradeableProgram {
     pub program_id: String,
-    pub program_executable: bool,
-    pub program_lamports: u64,
     pub programdata_address: String,
```

Comment:
```suggestion
    pub program_data_address: String,
```

Reply:
This is referred to as ProgramData everywhere so I'm going to keep it.